cypress/features/regression/dialogFullScreen.feature

```diff
@@ -86,4 +86,19 @@ Feature: Dialog Full Screen component
       And clear all actions in Actions Tab
       And I open component preview
     When I click closeIcon in IFrame
-    Then cancel action was called in Actions Tab
\ No newline at end of file
+    Then cancel action was called in Actions Tab
+
+  @positive
+  Scenario: Verify that most top dialog is closed by pressing Esc key
```

Comment:
```suggestion
  Scenario: Verify that nested dialog is closed by pressing Esc key
```

Reply:
Please, change the description of steps to make TC more readable and understandable